src/components/activity-page/embeddable.tsx

```diff
@@ -93,6 +93,10 @@ export const Embeddable: React.ForwardRefExoticComponent<IProps> = forwardRef((p
     qComponent = <div>Content type not supported</div>;
   }
 
+  if (qComponent === undefined) {
```

Comment:
Can you add a comment here about how LARA handles this so there isn't a direct dependency on teacherEditionMode, and include a link to the code in there that uses a mutation observer which handles this.

This will be useful if we ever try to remove the teacher edition coupling.